icd/CMakeLists.txt

```diff
@@ -227,6 +227,13 @@ endif()
 ### PAL ########################################################################
 add_subdirectory(${XGL_PAL_PATH} ${PROJECT_BINARY_DIR}/pal)
 
+### XGL cache creator tool ####################################################
+if(ICD_BUILD_LLPC)
+    get_target_property(XGL_COMPILE_OPTIONS xgl COMPILE_OPTIONS)
+    get_target_property(XGL_COMPILE_DEFINITIONS xgl COMPILE_DEFINITIONS)
```

Comment:
These variables aren't used in this file.

Query these properties in file you need them:
 tools/cache_creator/CMakeLists.txt